lib/wings/valkyrie/persister.rb

```diff
@@ -53,6 +53,7 @@ def delete(resource:)
         af_object = ActiveFedora::Base.new
         af_object.id = resource.alternate_ids.first.to_s
         af_object.delete
+        resource
```

Comment:
I _think_ this is okay, but it changes the expected returning value.  I'm thinking of the AR pattern (e.g. `#delete` returns a boolean).  However, this is not at the same place.

I also suspect that nothing upstream is checking on the return value of this method.

Thoughts?

Reply:
yes, this brings the return value in line with Valkyrie's interface. see https://github.com/samvera/valkyrie/blob/master/lib/valkyrie/persistence/postgres/persister.rb#L57

i think the missing test coverage is due to the valkyrie shared examples not being run (since order and several days types are unsupported). I'm running those tests now regularly as i work on order support. this maybe risks regression, but nothing in the hyrax code base seems to count on this return type, so it feels like a small risk(?).

the old return value was the AF object.